nodejs/week1/avg.js

```diff
@@ -0,0 +1,27 @@
+/*/create a file named avg.js which will calculate the 
+average of the numbers provided as command line arguments./*/
+const input = process.argv.slice(2).map(Number); 
+
+export default function getAverage(...numbers) { 
+    let sum = 0;
+    for (let i = 0; i < numbers.length; i++) {
+        sum += numbers[i]; 
+    }
+    const average = sum / numbers.length;
+    return average;
+};
+
+console.log(getAverage(...input));
+
+// Optional improvements and considerations:
+//filter the values we pass thru getaverage 
+const numbers = input.filter(num => !isNaN(num)); 
```

Comment:
Nice! That is a great way to avoid using values that are not numbers in our summing logic. 🧠  In this case, if the user passed values that are not numbers, they would just get ignored.